### PR TITLE
Sync with ballerina 0.991.0

### PR DESCRIPTION
**Pull request by Biruntha**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: Ballerina.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [project]
 org-name = "wso2"
-version = "0.8.2"
+version = "0.8.4"
```
Comment (Manuri) [inline Ballerina.toml]: Why aren't we upgrading the version to 0.8.3?

Comment (Biruntha) [reply Ballerina.toml]: latest version of this connector in BC is 0.8.3. That's why I put 0.8.4

File: README.md [hunk 2]
```diff
@@ -23,10 +23,16 @@ bigquery2:BigqueryConfiguration bigqueryConfig = {
...
             config: {
-                accessToken: testAccessToken,
-                clientId: testClientId,
-                clientSecret: testClientSecret,
-                refreshToken: testRefreshToken
+                grantType: http:DIRECT_TOKEN,
+                config: {
+                    accessToken: config:getAsString("ACCESS_TOKEN"),
+                    refreshConfig: {
+                        refreshUrl: REFRESH_URL,
```
Comment (MaryamZi) [inline README.md]: Shouldn't this be 
```suggestion
                        refreshUrl: bigquery2:REFRESH_URL,
```
Please check all places.

File: bigquery2/bigquery_endpoint.bal [hunk 3]
```diff
@@ -490,10 +490,6 @@ public function Client.init(BigqueryConfiguration bigqueryConfig) {
     http:AuthConfig? authConfig = bigqueryConfig.clientConfig.auth;
     if (authConfig is http:AuthConfig) {
```
Comment (MaryamZi) [inline bigquery2/bigquery_endpoint.bal]: Do we need this?

Comment (MaryamZi) [reply bigquery2/bigquery_endpoint.bal]: We will be overwriting what the user has specified if we do this right?

**Pull request merged**